cmd/kube-scheduler/app/server.go

```diff
@@ -287,6 +287,11 @@ func NewSchedulerConfig(s schedulerserverconfig.CompletedConfig) (*scheduler.Con
 		storageClassInformer = s.InformerFactory.Storage().V1().StorageClasses()
 	}
 
+	// If not set by the user, this should have been defaulted by component config defaulting
+	if s.ComponentConfig.BindTimeoutSeconds == nil {
```

Comment:
This check belongs to a recently added file: https://github.com/kubernetes/kubernetes/blob/be11540775e36b1f1acdd6cb43500c1717af0ad4/pkg/scheduler/apis/config/validation/validation.go#L28